paddlenlp/transformers/transformer/modeling.py

```diff
@@ -7,10 +7,27 @@
 import paddle.nn.functional as F
 from paddle.fluid.layers.utils import map_structure
 
+__all__ = [
+    "position_encoding_init", "WordEmbedding", "PositionalEmbedding",
+    "CrossEntropyCriterion", "TransformerDecodeCell",
+    "TransformerBeamSearchDecoder", "TransformerModel", "InferTransformerModel"
+]
+
 
 def position_encoding_init(n_position, d_pos_vec, dtype="float32"):
     """
-    Generate the initial values for the sinusoid position encoding table.
+    Generate the initial values for the sinusoidal position encoding table.
+    This method follows the implementation in tensor2tensor, but is slightly
+    different from the description in "Attention Is All You Need".
+
+    Args:
+        n_position (`int`): 
+            The largest position for sequences, that is, the maximum length
+            of source or target sequences.
+        d_pos_vec (`int`): 
+            The size of positional embedding vector. 
```

Comment:
The size of positional embedding vector. 
->
Dimensionality of positional embedding vector.